libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/FlyoutObjectDetection.tsx

```diff
@@ -201,11 +299,11 @@ export class FlyoutObjectDetection extends React.Component<
                     </Stack>
                   </Stack.Item>
```

Comment:
If I'm remembering correctly, I think the line limit is 300 for files like this :(. Might need to break up this file into separate components (maybe add helper functions to a common file)